Networking/Apiary and API call mocking.md

```diff
@@ -0,0 +1,179 @@
+                        "votes": 1
+                    }
+                ]
+            }
+        ]
+
+### POST some information [POST]
+
++ Response 201 (application/json)
+
+    + Headers
+
+            Location: /collection/2
+
+    + Body
+
+            {
+                "information": "Favourite programming language?",
+                "published_at": "2015-08-05T08:40:51.620Z",
+                "choices": [
+                    {
+                        "choice": "Swift",
+                        "votes": 2
+                    } 
+                ]
+            }
+
+```
+Apiary usually uses `API Blueprint`, a powerful high-level API description language for web APIs, more details on that can be found [here](https://apiblueprint.org).
+
+
+## Why Apiary ?
+
+Here are some of reasons why you might want to add apiary to your project: 
+
+* **Easy to learn**, not just for developers, but for testers also
+
+* **Multiplatform sync**, once written API can be used for iOS, Android or Web 
+
+* **Time-saving**, takes only **1** person to setup mock for multiple platforms
+
+* **Documentation!** As you write your responses, apiary will automatically generate great documentation
+
+* **Traffic inspector** - one click away from all of the API traffic
+
+* **Easy changes**, no need to rebuild project to apply changes, just edit apiary
+
+* **Improves testability**, QA can go through most test cases without support of the backend
+
+* Because it is **awesome**
+  
+
+## Know your limits
+
+Here are some of the limitations of apiary:
+
+* **You need network connection** - no internet, no mocking
+
+* **Backend Speed**, Apiary is usually very fast so developers might get a wrong feeling about some response times
+
+* **Changes**, if the app has to update some value, it might be a bit harder to mock
+
+* **Editing by hand**, all the mocks are just one large JSON file
+
+* **Merging**, multiple people editing at the same time
+
+* **Privacy**, responses are publically available, clients might dislike that, this is the reason why you will need to ask your clients to use apiary
+
+## Apiary setup
+
+Step 1 
+
+* Visit [Apiary.io](https://apiary.io), sign up and enter your information.
+
+Step 2
+
+* Give your API a name, your project name should be fine, and pick API format; You can choose between `Api Blueprint` and `Swagger`. If you are not sure which format to pick, just go with the default.
+
+Step 3
+
+* Already Done? Almost. Now navigate to `Inspector`. Here you can see all of the latest requests, however at the moment that will probably be empty. Right now you should see something like this: `Listening at https://some-private-link-here.com` . Now copy that link!
+
+Step 4
+
+* Go back to `Editor`. Replace `HOST: https://polls.apiblueprint.org/` with `HOST: https://that-private-link-you-just-copied.com`
+
+At this moment setup is done and you should end up with something like this:
+ 
+![Default](https://i.imgur.com/BHfNWfT.png)
```

Comment:
No need for 3rd party storing service, just put images in `resources` folder (or images, not sure at the moment)) and reference them via relative path (open some of bitrise tutorials and see how is done there)